documentation/Definition of DONE.md

```diff
@@ -9,15 +9,16 @@
 * I rather made many small steps than a few big steps.
 * I used images and code examples in my tutorial which were provided in the devonfw documentation. Plain text gets boring after time.
 * I used an organized structure in my tutorial.
-* All the things that the user should do actively, I have always written with function calls and steps . I have not given any instructions for action in text form.
+* Things that the user should do actively, I have written with function calls and steps. I have not given any instructions for actions in text form.
 * I proofread my tutorial:
      1. Are the steps available and in the correct order? 
      1. Are there any spelling mistakes? 
      1. Does everything make sense?
 * I ran through the latest update of my tutorial in Katacoda-Dev and checked if everything was working as intended BEFORE requesting a review.
 
 # **Technical**
-* The files that were used in `nextKatacodaStep `are available in `.asciidoc` files and do not contain Katacoda Syntax. `nextKatacodaStep` is almost never needed.
+* The files that were used in `nextKatacodaStep `are available in `.asciidoc` files and do not contain Katacoda Syntax.
+* The name of the function `nextKatacodaStep ` has been changed to make it clearer that there is no functional content allowed. Now it is called `displayContent ` 
```

Comment:
Why don't you just use the name `displayContent`. I think it is a bit confusing when you first mention `nextKatacodaStep` and then `displayContent`. We should remove `nextKatacodaStep` as it is not used anymore.

Reply:
Then I guess I should remove the entire sentence, as the sole purpose of mentioning nextKatacodaStep here was to remind the tutorial-makers to not use it. But it was renamed entirely so we do not need to mention it at all.

Reply:
Yes, just mention `displayContent` and that no Katacoda syntax should be used inside files used in this step